### PR TITLE
Minor formatting fix

### PR DESCRIPTION
**Pull request by sam-artuso**

### CODE REVIEW

File: pages/en/lb2/Environment-specific-configuration.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ LoopBack will always load the following configuration files, if they exist:
...
-* `server/config.{_env_``}.json/js`
-* `server/datasources.{_env_``}.json/js`
-* `server/model-config.{_env_``}.json/js`
-* `server/middleware.{_env_``}.json/js`
-* `server/component-config.{_env_``}.json/js `
+* `server/config.{_env_}.json/js`
+* `server/datasources.{_env_}.json/js`
+* `server/model-config.{_env_}.json/js`
+* `server/middleware.{_env_}.json/js`
+* `server/component-config.{_env_}.json/j`
```
Comment (dhmlau) [inline pages/en/lb2/Environment-specific-configuration.md]: i think there's a typo, should end with `js`

**Review by dhmlau — COMMENTED** [review]
@sam-artuso , thanks for submitting the PR.  Please see my review comment.

File: pages/en/lb2/Environment-specific-configuration.md (hunk 1 above)
Comment (sam-artuso) [reply pages/en/lb2/Environment-specific-configuration.md]: Fixed.

**Comment by dhmlau** [comment]
Looks like the PR is out of date. Let me rebase for you.

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
Thanks @sam-artuso.  Your PR has landed. 🎉

**Pull request merged**